patient-journey/patient-journey-quickstart/README.md

```diff
@@ -0,0 +1,24 @@
+# Patient Journey Quickstart
+
+This example shows how to use Neo4j Graph Analytics within Snowflake to analyze life sciences data, focusing on patient journey analytics. Specifically, this example shows how to:
+
+1. __Segment patient into cohorts__ based on medical history (graph algos: node similarity & louvain community detection)
+
+## Dataset
+We used [Synthea](https://github.com/synthetichealth/synthea) to simulate realistic patient data. 
+
+
+## Prerequisites
+
+### 1) Get Source Data
```

Comment:
The only one "step" in prerequisites - I think "1)" can be deleted

Reply:
Fixed this.